### PR TITLE
Minor update to include video in initial steps

### PR DESCRIPTION
**Pull request by Felonious-Spellfire**
## What does this PR do?

Updating content to include video in the initial steps

### Does your PR introduce any breaking changes? If yes, please list them.

None

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [X] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [X] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @rohitgr7 @Felonious-Spellfire

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

File: docs/source-app/workflows/encrypted_secrets/secrets.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Is your App using data or values (for example: API keys or access credentials) that you don't want to expose in your App code? If the answer is yes, you'll want to use Secrets. Secrets are encrypted values that are stored in the Lightning.ai database and are decrypted at runtime.
+
+.. tip::
+	For non-sensitive configuration values, we recommend using :ref:`plain-text Environment Variables <environment_variables>`.
+
+***************
+What did we do?
+***************
+
+When a Lightning App (App) **runs in the cloud**, a Secret can be exposed to the App using environment variables.
```
Comment (rohitgr7) [inline docs/source-app/workflows/encrypted_secrets/secrets.rst:16]: can you add an example to show how to access them?

File: docs/source-app/workflows/encrypted_secrets/secrets.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+The value of the Secret is encrypted in the Lightning.ai database, and is only decrypted and accessible to
+LightningFlow (Flow) or LightningWork (Work) processes in the cloud (when you use the ``--cloud`` option running your App).
```
Comment (rohitgr7) [inline docs/source-app/workflows/encrypted_secrets/secrets.rst:18]:
```suggestion
When a Lightning App (App) **runs in the cloud**, a Secret can be exposed to the App using environment variables.
The value of the Secret is encrypted in the Lightning.ai database, and is only decrypted and accessible in the
LightningFlow (Flow) or LightningWork (Work) processes in the cloud (when you use the ``--cloud`` option running your App).
```

File: docs/source-app/workflows/encrypted_secrets/secrets.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
+.. _secrets:
+
+#################
+Encrypted Secrets
+#################
+
+Is your App using data or values (for example: API keys or access credentials) that you don't want to expose in your App code? If the answer is yes, you'll want to use Secrets. Secrets are encrypted values that are stored in the Lightning.ai database and are decrypted at runtime.
```
Comment (rohitgr7) [inline docs/source-app/workflows/encrypted_secrets/secrets.rst:7]:
```suggestion
Is your App using data or values (for example: API keys or access credentials) that you don't want to expose in your App code? If the answer is yes, you'll want to use Secrets. Secrets are encrypted values stored in the Lightning.ai database and decrypted at runtime.
```

File: docs/source-app/workflows/encrypted_secrets/encrypt_secrets.rst [hunk 4]
```diff
@@ -0,0 +1,14 @@
+.. _encrypt_secrets:
+
+########################
+Encrypt your credentials
+########################
+
+Is your App using data or values (for example: API keys or access credentials) that you don't want to expose in your App code? If the answer is yes, you'll want to use Secrets. Secrets are encrypted values that are stored in the Lightning.ai database and are decrypted at runtime.
```
Comment (rohitgr7) [inline docs/source-app/workflows/encrypted_secrets/encrypt_secrets.rst:7]:
```suggestion
Is your App using data or values (for example: API keys or access credentials) that you don't want to expose in your App code? If the answer is yes, you'll want to use Secrets. Secrets are encrypted values stored in the Lightning.ai database and decrypted at runtime.
```

**Review by rohitgr7 — APPROVED** [review]

File: docs/source-app/workflows/encrypted_secrets/secrets.rst (hunk 1 above)
Comment (Felonious-Spellfire) [reply docs/source-app/workflows/encrypted_secrets/secrets.rst:16]: The example is in the section below.

**Review by carmocca — APPROVED** [review]
I don't think setup.py should be set to execution. Hope reverting that doesn't break anything :)

**Comment by justusschock** [comment]
Closed because of conflicts with docs revamp

**Pull request closed without merging**